tests/test_run_feature_vector.py

```diff
@@ -6,61 +6,64 @@
 import pandas as pd
 import pytest
 from dictdiffer import diff
-
-
-TEST_OUTPUT_DIR = "/allen/aibs/informatics/module_test_data/ipfx/test_feature_vector"
-
-
-@pytest.mark.skip(
-    reason=(
-        "this test relies on a lims query for now-unsupported NWB1 data, "
-        "it must be replaced with direct access to a well-known NWB2 file."
-    )
-)
-@pytest.mark.requires_lims
-@pytest.mark.slow
-@pytest.mark.skipif(
-    not os.path.exists(TEST_OUTPUT_DIR), 
-    reason="unable to read expected data"
-)
-def test_feature_vector_extraction(tmpdir_factory):
-
-    temp_output_dir = str(tmpdir_factory.mktemp("feature_vector"))
-    test_output_dir = TEST_OUTPUT_DIR
-
-    features = [
-        "first_ap_v",
-        "first_ap_dv",
-        "isi_shape",
-        "psth",
-        "inst_freq",
-        "spiking_width",
-        "spiking_peak_v",
-        "spiking_fast_trough_v",
-        "spiking_threshold_v",
-        "spiking_upstroke_downstroke_ratio",
-        "step_subthresh",
-        "subthresh_norm",
-        "subthresh_depol_norm",
-        ]
-
-    run_feature_vector_extraction(ids=[500844783, 509604672],
-                                  output_dir=temp_output_dir,
-                                  data_source="lims",
-                                  output_code="TEMP",
-                                  project=None,
-                                  output_file_type="npy",
-                                  sweep_qc_option="none",
-                                  include_failed_cells=True,
-                                  run_parallel=False,
-                                  ap_window_length=0.003
-                                  )
-
-    for feature in features:
-        test_data = np.load(os.path.join(test_output_dir, "fv_{:s}_TEMP.npy".format(feature)))
-        temp_data = np.load(os.path.join(temp_output_dir, "fv_{:s}_TEMP.npy".format(feature)))
-
-        assert np.allclose(test_data, temp_data)
+import unittest
+import shutil, tempfile
+
+path_to_current_file = os.path.realpath(__file__)
+current_directory = os.path.split(path_to_current_file)[0]
+
+TEST_OUTPUT_DIR = os.path.join(current_directory, "data/feature_vector")
+
+nwb2_file1 = os.path.join(current_directory, "data/Vip-IRES-Cre;Ai14(IVSCC)-226110.03.01.nwb")
```

Comment:
looks like you are not using git-lfs and are committing all the test files to git

Reply:
those are committed to git-lfs and all files under the "tests/data" folder